installer/x86_64/install.sh

```diff
@@ -51,6 +51,10 @@ CONSOLE_SPEED=9600
 
 # Get platform specific linux kernel command line arguments
 ONIE_PLATFORM_EXTRA_CMDLINE_LINUX=""
+
+# Default var/log device size in MB
+VAR_LOG_SIZE=4000
```

Comment:
Arbitrary sizre can be placed in corresponding platform file, e. g. installer/x86_64/platforms/x86_64-mlnx_x86-r5.0.1400
I don't think that we can disable it, because during build I have to create a mount entry in /etc/fstab
doesn''t configurable size cover all cases?